### ISSUE TITLE
JAWS is not Reading the expanded content.

### ISSUE THREAD

**Issue by srikanthmadasu** [GSA/sam-design-system#1157]
Components Involved
sdsCollapse

Expected Behavior
How you expect the listed component(s) to function
JAWS should be reading the content after its expanded

Actual Behavior
How the listed component(s) actually function
JAWS does not read the content automatically. How ever when we hit down arrow it starts reading it.

Your Environment
Angular Version: X.Y.Z
@gsa-sam packages version: X.Y.Z (Please list individually if all packages are not same version)
Angular Version: "^13.3.11"
GSA Packages:
@gsa-sam/components@13.0.5
@gsa-sam/sam-material-extensions@13.0.5
@gsa-sam/sam-formly@13.0.5
@gsa-sam/layout@13.0.3
@gsa-sam/layouts@13.0.0-beta-0
@gsa-sam/ngx-uswds@13.0.0-beta.1
@gsa-sam/ngx-uswds-icons@13.0.0-beta.2
@gsa-sam/sam-styles@0.0.150
@gsa-sam/shared@13.1.10

Ticket Number
Please provide your JIRA ticket number that is dependent on this fix. If there is none, please create
one to validate the issue once the fix has been released. DO NOT provide the url, please ONLY
provide the ticket number.
IAEMOD-12711

Additional context
Screenshots / recordings, minimal reproduction on stackblitz, steps to reproduce, code snippets, etc.
![image](https://user-images.githubusercontent.com/24394017/230669523-a2006a5a-a109-4ceb-803d-e3a465253ff7.png)

SampleCode:
![image](https://user-images.githubusercontent.com/24394017/230668756-7696cc57-1716-4cb9-b2fe-c30dd8c75f8f.png)

**Comment by cwolf10** [comment]
Closing as there is no issue with the component. Feel free to follow up if there is some additional info that can be provided.
